content/en/dashboards/_index.md

```diff
@@ -96,19 +99,37 @@ An organization's most popular dashboard displays five popularity bars. All othe
 
 ## New dashboard
 
-To create a dashboard, click on the **New Dashboard +** button in the upper right. Enter a dashboard name and choose between creating a [timeboard][3] or a [screenboard][4].
+<div class="alert alert-warning">The new dashboard functionality is in public beta.</div>
```

Comment:
we can probably remove this banner- the new layout is in beta, but "new dashboard" itself is not in beta